http4s/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/model.scala

```diff
@@ -27,7 +29,21 @@ object model {
     */
   final case class SplitBatchResult(goodBatches: List[List[Json]], failedBigEvents: List[Json])
 
-  final case class CollectorConfig(
-    paths: Map[String, String]
+  final case class CookieConfig(
+    enabled: Boolean,
+    name: String,
+    expiration: FiniteDuration,
+    domains: Option[List[String]],
```

Comment:
Is there any significant difference between `None` and `Some(Nil)`?  Did you consider making this a non-optional `List[String]` with a default of `Nil`?

Reply:
I don't think there is a difference between `None` and `Some(Nil)`. We can use `List[String]` without wrapping it to `option` I guess. I will make necessary changes.